fpx-lib/src/api/handlers/spans.rs

```diff
@@ -101,3 +101,40 @@ impl From<DbError> for ApiServerError<SpanListError> {
         ApiServerError::CommonError(CommonError::InternalServerError)
     }
 }
+
+#[tracing::instrument(skip_all)]
+pub async fn span_delete_handler(
+    State(store): State<BoxedStore>,
+    Path((trace_id, span_id)): Path<(String, String)>,
+) -> Result<StatusCode, ApiServerError<SpanDeleteError>> {
+    let tx = store.start_readonly_transaction().await?;
+
+    hex::decode(&trace_id)
+        .map_err(|_| ApiServerError::ServiceError(SpanDeleteError::InvalidTraceId))?;
+    hex::decode(&span_id)
+        .map_err(|_| ApiServerError::ServiceError(SpanDeleteError::InvalidSpanId))?;
+
+    let _ = store.span_delete(&tx, &trace_id, &span_id).await?;
```

Comment:
is this marked `#[must_use]` that this is bound to `_`?

Reply:
No it isn't, will remove the assignment